src/libs/ifc/xml/vabstractpattern.cpp

```diff
@@ -78,143 +78,143 @@
+const QString VAbstractPattern::TagSizes                = QStringLiteral("sizes");
+const QString VAbstractPattern::TagData                 = QStringLiteral("data");
+const QString VAbstractPattern::TagPatternInfo          = QStringLiteral("patternInfo");
+const QString VAbstractPattern::TagPatternName          = QStringLiteral("patternName");
+const QString VAbstractPattern::TagPatternNum           = QStringLiteral("patternNumber");
+const QString VAbstractPattern::TagCustomerName         = QStringLiteral("customer");
+const QString VAbstractPattern::TagCompanyName          = QStringLiteral("company");
+const QString VAbstractPattern::TagPatternLabel         = QStringLiteral("patternLabel");
+const QString VAbstractPattern::TagGrainline            = QStringLiteral("grainline");
+const QString VAbstractPattern::TagPath                 = QStringLiteral("path");
+const QString VAbstractPattern::TagNodes                = QStringLiteral("nodes");
+const QString VAbstractPattern::TagNode                 = QStringLiteral("node");
+
+const QString VAbstractPattern::AttrName                = QStringLiteral("name");
+const QString VAbstractPattern::AttrVisible             = QStringLiteral("visible");
+const QString VAbstractPattern::AttrObject              = QStringLiteral("object");
+const QString VAbstractPattern::AttrTool                = QStringLiteral("tool");
+const QString VAbstractPattern::AttrType                = QStringLiteral("type");
+const QString VAbstractPattern::AttrLetter              = QStringLiteral("letter");
+const QString VAbstractPattern::AttrAnnotation          = QStringLiteral("annotation");
+const QString VAbstractPattern::AttrOrientation         = QStringLiteral("orientation");
+const QString VAbstractPattern::AttrRotationWay         = QStringLiteral("rotationWay");
+const QString VAbstractPattern::AttrTilt                = QStringLiteral("tilt");
+const QString VAbstractPattern::AttrFoldPosition        = QStringLiteral("foldPosition");
+const QString VAbstractPattern::AttrQuantity            = QStringLiteral("quantity");
+const QString VAbstractPattern::AttrOnFold              = QStringLiteral("onFold");
+const QString VAbstractPattern::AttrDateFormat          = QStringLiteral("dateFormat");
+const QString VAbstractPattern::AttrTimeFormat          = QStringLiteral("timeFormat");
+const QString VAbstractPattern::AttrArrows              = QStringLiteral("arrows");
+const QString VAbstractPattern::AttrNodeReverse         = QStringLiteral("reverse");
+const QString VAbstractPattern::AttrNodeExcluded        = QStringLiteral("excluded");
+const QString VAbstractPattern::AttrNodeNotch           = QStringLiteral("passmark");
+const QString VAbstractPattern::AttrNodeNotchType       = QStringLiteral("passmarkLine");
+const QString VAbstractPattern::AttrNodeNotchSubType    = QStringLiteral("passmarkAngle");
+const QString VAbstractPattern::AttrNodeShowSecondNotch = QStringLiteral("showSecondPassmark");
```

Comment:
?
Should "showSecondPassmark" be "showSecondNotch"?

Reply:
Not yet. This is a schema element. I will make that change when I add the notch improvements. It saves having to do 2 schemas.

Reply:
ok super